pkg/TIPA.py

```
@@ -164,587 +165,498 @@ def __init__(self, img, mouse_pos, display_info_init, gui_queue):
+            logging.debug(corrected_text)
+
+        rep = {
+            " ": "+", "$": "", "/": "_", r"[\/\\\n|_]*": "_", "muzzle": "muzzlebrake", "brake": "", "7.6239": "7.62x39",
+            "5.5645": "5.56x45", "MPS": "MP5", "MP3": "MP5", "Flash hider": "Flashhider", "]": ")", "[": "(", "sung": "sunglasses",
+            "X/L": "X_L", "Tactlcal": "Tactical", "AK-103-762x39": "", "l-f": "l_f", "away": "", "MK2": "Mk.2", '"Klassika"': "Klassika",
+            "^['^a-zA-Z_]*$": "%E2%80%98", "RUG": "RDG", "AT-2": "AI-2", "®": "", "§": "5", "__": "_", "___": "", "xX": "X", "SORND": "50RND",
+            "Bastion dust cover for AK": "Bastion_dust_cover_for_%D0%B0%D0%BA", "PDC dust cover for AK-74": "PDC_dust_cover_for_%D0%B0%D0%BA-74",
+            "XLORUNO-VM": "KORUND-VM", "SURVIZ": "SURV12", "TOR": "Vector 9x19", "SPLIN": "SPLINT", "DSCRX": "D3CRX", "SSO": "SSD",
+            "((": "(", "))": ")"
+        }
+        rep = dict((re.escape(k), v) for k, v in rep.items())
```

Ruff (0.8.0)

Unnecessary generator (rewrite as a `dict` comprehension)

Rewrite as a `dict` comprehension

(C402)